core/src/tile/tileManager.h

```diff
@@ -26,6 +27,10 @@ class View;
  */
 class TileManager {
 
+    const static size_t MAX_WORKERS = 2;
+    const static size_t MAX_DOWNLOADS = 4;
+    const static size_t DEFAULT_CACHE_SIZE = 32*1024*1024; // 32 MB
```

Comment:
@karimnaaji I'm not quite sure if I understood - but yes, the tiles in the cache are not updated or drawn as long as they are not visible